python/core/symbology-ng/qgsstyle.sip

```diff
@@ -208,8 +208,27 @@ class QgsStyle : QObject
     //! Changes ramp's name
     bool renameColorRamp( const QString& oldName, const QString& newName );
 
-    //! Creates a temporary memory database
+    /** Creates an on-disk database
+     *
+     *  This function creates a new on-disk permanent style database.
+     *  \return returns the success state of the database creation
+     *  @see createMemoryDB()
+     */
+    bool createDB( const QString& filename );
```

Comment:
I think that should be `createDb`, acronyms have been updated to this style for 3.0